anime-list-master.xml

```diff
@@ -4573,7 +4573,7 @@
   <anime anidbid="1236" tvdbid="94701" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Devilman</name>
   </anime>
-  <anime anidbid="1237" tvdbid="unknown" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="1237" tvdbid="296106" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
```

Comment:
ID doesn't exist
https://thetvdb.com/dereferrer/series/296106

Reply:
My bad on this one corrected with https://thetvdb.com/series/396707-. Seems like there is a error in TVDB as the link won't work without the -.